addon/-private/system/model/model.js

```diff
@@ -1,54 +1,29 @@
 import Ember from 'ember';
-import { assert, deprecate, warn } from "ember-data/-private/debug";
-import { PromiseObject } from "../promise-proxies";
-import Errors from "../model/errors";
+import { assert, deprecate } from 'ember-data/-private/debug';
+import { PromiseObject } from '../promise-proxies';
+import Errors from '../model/errors';
 import isEnabled from '../../features';
 import RootState from '../model/states';
+<<<<<<< 2edaf160a8a437cc989b2e6c980cc9c7d525c2c2
```

Comment:
ugh, not sure how these crept it during my rebase (silly editor telling me file had been fixed)